docs/module/buff.md

```diff
@@ -0,0 +1,153 @@
+# RM_Buff 使用手册
+
+---
+
+## 一、TODO LIST
+
+---
+
+- [x]  文件夹架构
+  - [x]  xml
+  - [x]  object
+  - [x]  fanarmor
+  - [x]  fanblade
+  - [x]  target
+  - [x]  circle
+- [x]  接口函数
+- [ ]  Test 文件
+- [ ]  Release
+- [ ]  Predict
+- [ ]  DEBUG (I'm a typical BUG manufacturer /(ㄒoㄒ)/~~)
+- [ ]  Auto Control
+
+## 二、使用说明
+
+---
+
+### 步骤一：包含头文件
+
+- 包含头文件`RM_Buff.hpp`
```

Comment:
If we don't finish the documentation, we'd better add it later, because such things can cause misunderstandings like this:

```suggestion
- 包含头文件`module/basic_buff.hpp`
```

Reply:
Modify later